Platform/RaptorlakeBoardPkg/Rplps/Microcode/Microcode.inf

```diff
@@ -0,0 +1,18 @@
+## @file
+#
+#  Copyright (c) 2020 - 2023, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#
+##
+
+[Defines]
+  INF_VERSION          = 0x00010005
+  BASE_NAME            = Microcode
+  FILE_GUID            = 197DB236-F856-4924-90F8-CDF12FB875F3
+  MODULE_TYPE          = USER_DEFINED
+  VERSION_STRING       = 1.0
+
+[Sources]
+m_e0_b06a2_00004121.mcb    # RPL-PS
+m_80_906a3_00000429.mcb    # ADL-P/M Lx/Rx
```

Comment:
@bejeanmo do we need this m_80_906a3_00000429.mcb?

Reply:
Thanks. I thought I had removed these already. Should be fixed now.